scripts/protoc-swagger-gen.sh

```diff
@@ -2,16 +2,31 @@
 
 set -eo pipefail
 
+# check if there is protoc binary
+if ! command -v protoc &> /dev/null
+then
+  echo "protoc could not be found. Please install http://google.github.io/proto-lens/installing-protoc.html"
```

Comment:
The version of this has to match the version of our `github.com/cosmos/gogoproto` dependency? Because if that's the case, we better recommend installing a version too.

Reply:
for now, I can protoc with current github.com/cosmos/gogoproto

Reply:
should it becomes incompatible in the future, I will add recommended protoc version

Reply:
Can we resolve this comment? :)